apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerDbType.ValueConversion.cs

```diff
@@ -173,6 +173,13 @@ internal Value ToProtobufValue(object value, SpannerConversionOptions options)
                         StringValue = XmlConvert.ToString(Convert.ToDateTime(value, InvariantCulture), XmlDateTimeSerializationMode.Utc)
                     };
                 case TypeCode.Date:
+                    if (value is DateTime date && date.Kind == DateTimeKind.Local)
+                    {
+                        // If the DateTime is local the value should be changed to unspecified
+                        // before serializing it as UTC to prevent accidentally changing to a
+                        // different date.
+                        value = new DateTime(date.Year, date.Month, date.Day);
```

Comment:
It *might* be clearer to have:

```csharp
value = DateTime.SpecifyKind(date.Date, DateTimeKind.Unspecified);
```

See what you think.

Reply:
Yes, that does seem more appropriate for what the intention is here. Changed.